### DISCUSSION TITLE
does not have the correct permissions to execute \`Remove Reaction\` -- for already removed reaction

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#110517] [Mobile] [answered]
GitHub Mobile beta for Android 

https://github.com/google/temporian/pull/271#issuecomment-1970841347

> jsoref does not have the correct permissions to execute \`Remove Reaction\`
![Screenshot_20240229-101626.png](https://github.com/community/community/assets/2119212/408c8edf-c03d-4741-8c31-5ef5f8c8ad94)

Steps:
1. Open thing in GitHub Mobile
2. Add reaction in GitHub Mobile
3. On another device, remove reaction
4. On GitHub Mobile, tap reaction to remove it (because you forgot about the previous step)

Expected results:
* Don't show a scary error for this case
* Either silently update the display to reflect the user's intent to have the reaction removed has succeeded, or
* Proactively update the reactions list so that it isn't visible so that users won't be able to try to tap a 👻 reaction (unless they're trying to race multiple devices)

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref I filed this for the team to triage. Thanks for writing in!
